### PR TITLE
Add card emoji tests in Apache commons sets

### PR DESCRIPTION
**Pull request by aqsa505**
Signed-off-by: Aqsa Malik <aqsa.malik@bnymellon.com>

### CODE REVIEW

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 1]
```diff
@@ -99,6 +100,9 @@ public void deal()
         Set<Card> jdkHand = this.jdkDeck.deal(jdkShuffle, 5);
         Set<Card> acHand = this.acDeck.deal(acShuffle, 5);
         Assertions.assertEquals(jdkHand, acHand);
+        Assertions.assertEquals(
+                "|3♦|, |5♥|, |6♥|, |3♣|, |Q♣|",
+                Iterate.makeString(acHand.stream().sorted().toList()));
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:105]: Use Apache Commons toString method the same as the Apache List solution.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 2]
```diff
@@ -107,6 +111,15 @@ public void shuffleAndDealHands()
         List<Set<Card>> jdkHands = this.jdkDeck.shuffleAndDeal(new Random(1), 5, 5);
         List<Set<Card>> acHands = this.acDeck.shuffleAndDeal(new Random(1), 5, 5);
         Assertions.assertEquals(jdkHands, acHands);
+        var hands = acHands.stream().map(each ->
+                        Iterate.makeString(each.stream().sorted().toList())).collect(Collectors.toSet());
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:115]: Use Apache Commons toString method the same as the Apache List solution.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 3]
```diff
@@ -117,6 +130,15 @@ public void dealHands()
         List<Set<Card>> jdkHands = this.jdkDeck.dealHands(jdkShuffled, 5, 5);
         List<Set<Card>> acHands = this.acDeck.dealHands(acShuffled, 5, 5);
         Assertions.assertEquals(jdkHands, acHands);
+        var hands = acHands.stream().map(each ->
+                Iterate.makeString(each.stream().sorted().toList())).collect(Collectors.toSet());
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:134]: Same comment.

File: deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 4]
```diff
@@ -99,6 +100,9 @@ public void deal()
         Set<Card> jdkHand = this.jdkDeck.deal(jdkShuffle, 5);
         Set<Card> acHand = this.acDeck.deal(acShuffle, 5);
         Assertions.assertEquals(jdkHand, acHand);
+        Assertions.assertEquals(
+                "|3♦|, |5♥|, |6♥|, |3♣|, |Q♣|",
+                Iterate.makeString(acHand.stream().sorted().toList()));
```
Comment (donraab) [inline deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:105]: Same comment.

File: deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 5]
```diff
@@ -107,6 +111,15 @@ public void shuffleAndDealHands()
         List<Set<Card>> jdkHands = this.jdkDeck.shuffleAndDeal(new Random(1), 5, 5);
         List<Set<Card>> acHands = this.acDeck.shuffleAndDeal(new Random(1), 5, 5);
         Assertions.assertEquals(jdkHands, acHands);
+        var hands = acHands.stream().map(each ->
+                Iterate.makeString(each.stream().sorted().toList())).collect(Collectors.toSet());
```
Comment (donraab) [inline deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:115]: Same comment.

File: deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java [hunk 6]
```diff
@@ -117,6 +130,15 @@ public void dealHands()
         List<Set<Card>> jdkHands = this.jdkDeck.dealHands(jdkShuffled, 5, 5);
         List<Set<Card>> acHands = this.acDeck.dealHands(acShuffled, 5, 5);
         Assertions.assertEquals(jdkHands, acHands);
+        var hands = acHands.stream().map(each ->
+                Iterate.makeString(each.stream().sorted().toList())).collect(Collectors.toSet());
```
Comment (donraab) [inline deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/ApacheCommonsDeckOfCardsAsSortedSetTest.java:134]: Same comment.

**Review by donraab — CHANGES_REQUESTED** [review]
Thank you for the contribution @aqsa505! A few changes requested inline.

**Comment by aqsa505** [comment]
Updated other tests with Apache Commons toString.

**Review by donraab — APPROVED** [review]
LGTM! Thanks for the contribution @aqsa505 !

**Pull request merged**